### ISSUE TITLE
Only some of the controller functions work

### ISSUE THREAD

**Issue by dusan-t** [r57zone/Half-Life-Alyx-novr#47]
Hi. I tried the driver in SteamVR Home and SteamVR empty enviroment, and some games that already have keyboard support (so not the best use case) and I can only get some of the controller functions to work. 
Left mouse button doesn't react at all and pressing the right mouse button only shows the laser pointer from the right controller but I can't move it. Also I can only move the left controller up and down, but not the right controller. Rotation doesn't work for either of the controllers. Right controller I can only move forward and backward together with the left controller and that's all the movement I can do with the right controller. Menu buttons (Enter, F9) also don't work. I can't even turn off or navigate the SteamVR dashboard.  Do you have any ideas on what I could do to fix this ? Thanks.

**Comment by r57zone** [comment]
Hi, have you installed other drivers? Have you tried with Half Life Alyx? In Half Life, Alyx you didn't changed two-handed to one-handed controls?

**Comment by dusan-t** [comment]
I don't think I have other 3rd party drivers installed. I am using this driver with Windows Mixed reality Headset in VR with key bindings for mobile and diy. Also I didn't add '+ vr_gg_gesture_pull 0' to game shortcuts but I don't think that's what's breaking it.
I haven't tried it with Alyx because I don't have it. I mainly wanted to use this driver for all the other VR games/experiences that require VR controllers.

**Comment by r57zone** [comment]
Have you disabled HMD in the Ini config file? If you have disabled and are not using other drivers, then the Windows Mixed Reality driver may be conflicting with this in some way.

**Comment by dusan-t** [comment]
The HMD is disabled in the .ini file. I was also thinking that WMR might be conflicting with this driver as WMR is also a 3rd party driver, although an official one. Since WMR has it's own controllers perhaps SteamVR when it sees WMR headset is in some way expecting WMR controllers as well.

**Comment by r57zone** [comment]
To check, you can disconnect WMR and enable HMD in ini, try without WMR maybe everything will be fine.

**Comment by dusan-t** [comment]
I tried before with HMD enabled in the .ini file but with WMR installed, I don't know if WMR can be disabled or only uninstalled, because the WMR is the base and SteamVR is the layer on top of it. In order for WMR to work with SteamVR the SteamVR driver for WMR has to be installed first (from the Steam store). So when I tried with HMD enabled in the .ini I was presented with the SteamVR setup (HMD, controllers, room) but when it got to the stage where I was asked to point the controller in the direction of my monitor and press and hold the trigger button on the controller I tried pressing and holding different buttons but couldn't proceed. So that approach didn't work. But thank you anyway for taking the time to respond.
